### PR TITLE
feat: fts examples

### PR DESCRIPTION
**Pull request by shyam-cb**

### CODE REVIEW

File: controllers/hotel_controller.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+func NewHotelController(airlineService services.IHotelService) *HotelController {
+	return &HotelController{
+		HotelService: airlineService,
+	}
+}
+
+// SearchByName
+//
+// @Summary Search by hotel name
+// @Description Search for hotels based on their name.
```
Comment (prajwal-pai77) [inline controllers/hotel_controller.go:24]: Can you please add more description like other routes. For eg: Code and method

File: controllers/hotel_controller.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+// @Tags Hotel
+// @Produce json
+// @Param name query string true "name search"
```
Comment (prajwal-pai77) [inline controllers/hotel_controller.go:27]: Can u please add an example like in other routes

File: controllers/hotel_controller.go [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+			return
+		}
+		context.JSON(http.StatusOK, hotels)
+	}
+}
+
+// Filter controller
+//
+// @Summary Fetch hotels with multiple filters
+// @Description Fetch hotels using various filters such as name, title, description, country, state and city.
```
Comment (prajwal-pai77) [inline controllers/hotel_controller.go:53]: Same as above, Can you please add more description like other routes. For eg: Code and method

**Review by prajwal-pai77 — COMMENTED** [review]
Minor swagger changes

File: controllers/hotel_controller.go (hunk 3 above)
Comment (shyam-cb) [reply controllers/hotel_controller.go]: done

File: controllers/hotel_controller.go (hunk 2 above)
Comment (shyam-cb) [reply controllers/hotel_controller.go:27]: done

**Pull request merged**